dev/io.openliberty.xmlBinding.3.0.internal.tools/jaxb_xjc.bnd

```diff
@@ -16,8 +16,8 @@ Bundle-Name: IBM JAXB XJC Tool
 
```

Comment:
Should this bundle name and symbolic name change to remove reference to jaxb?

Reply:
can do